File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        seen = {}
        k = 0
        for i, num in enumerate(nums):
            if num not in seen:
                nums[k] = num
                k+=1
            seen[num] = i
        return k

            
        

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 0;
        int target = nums[k++]; 
        for(int i=1; i<nums.length; i++){
            if(nums[i] != target){
                target = nums[i];
                nums[k] = nums[i];
                k++;
            }else{
                continue;
            }
        }
        return k;
    }
}